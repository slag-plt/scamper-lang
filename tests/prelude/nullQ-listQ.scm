(null? (list))
(null? (list "ab" "ba"))
(null? (list 1 2 3))
(null? (list ""))
(null? (list))
(list? (list))
(list? (list "ab" "ba"))
(list? (list 1 2 3))
(list? (list ""))
(list? (pair "hello" "world"))