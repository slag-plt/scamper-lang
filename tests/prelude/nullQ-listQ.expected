#t
#f
#f
#f
#t
#t
#t
#t
#t
#f
